cmd/flow_capture.go

```diff
@@ -315,17 +318,23 @@ func updateTable() {
 			tbl.Print()
 		}
 
-		if len(regexes) > 0 {
-			fmt.Printf("Live table filter: %s Press enter to match multiple regexes at once\n", regexes)
+		if len(keyboardError) > 0 {
+			fmt.Println(keyboardError)
```

Comment:
is it on purpose to use fmt.Print rather than the logger ? (I see log functions are used above)

Reply:
As soon as the table is drawn, I use only fmt to display properly without the log date & time

This message is displayed at the botom of the UI to explain why keyboard is not supported